index.html

```diff
@@ -0,0 +1,127 @@
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <button>See project</button>
+            </li>
+            <li class="container1">
+                    <div class="works-pic"><img src="Assets/files/Snapshoot3.png" alt=""></div>
+                    <h2 class="works-heading">Tonic</h2>
+                    <div class="canopy"><img src="Assets/files/Frame 2.png" alt=""></div>
+                    <p class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <button>See project</button>
+            </li>
+            <li class="container2">
+                    <div class="works-pic"><img src="Assets/files/Snapshoot4.png" alt=""></div>
+                    <h2 class="works-heading">Multi-Post Stories</h2>
+                    <div class="canopy"><img src="Assets/files/Frame 2.png" alt=""></div>
+                    <p class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                <button>See project</button>
+            </li>
+        </ul>
+    </section>
+    <section id="about-self">
+        <h3 class="aboutself-heading">About myself</h3>
+        <p class="aboutself-paragragh">Hello I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see  and have a project you need coded, don't hesitate to contact me.</p>
+        <h4 class="connect2">LET'S CONNECT</h4>
+            <ul id="social-icons">
+                <li><a href="#" class="icon"><img src="Assets/files/assets/github.png" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/Linkedin icon.png" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/Vector.jpg" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/twitter.png" alt="Loading"></a></li>
+                <li><a href="#" class="icon"><img src="Assets/files/assets/medium.png" alt="Loading"></a></li>
+            </ul>
+        <button class="resume">Get my resume</button>
+
+        <ul class="langues">
```

Comment:
Great job on your project! 👏👏👏 However, there's one issue that needs to be addressed. As the previous reviewer pointed out, only **li** tags should be direct children of the **ul** tag. The issue here is that you have `<div class="arrow"> <img src="Assets/files/arrow.png" alt=""></div>` and `<ul id="tech">` as direct children of `<ul class="langues">`.

To fix this issue, you can restructure your code like this:

```
<li class="languages1">
	<h3>Languages <img src="Assets/files/arrow.png" alt="Arrow icon"></h3>
  <ul id="tech">
     <li><button class="tag tag1"> Javascript</button></li>
     <li><button class="tag tag2"> HTML</button></li>
     <li><button class="tag tag3"> CSS</button></li>    
  </ul>
</li>
```

I hope this helps! Let me know if you have any questions. 😊